modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -377,6 +475,38 @@ namespace TypesTest
     };
 } // namespace TypesTest
 
+TEST_P(GAPI_Streaming, SmokeTest_One_Video_One_Const_Vector)
+{
+    cv::GMat in_m;
+    cv::GArray<int> in_v;
+    cv::GMat out_m = TypesTest::AddV::on(in_m, in_v) - in_m;
+
+    // Run pipeline
+    auto testc = cv::GComputation(cv::GIn(in_m, in_v), cv::GOut(out_m))
+                    .compileStreaming(cv::compile_args(cv::gapi::kernels<TypesTest::OCVAddV>()));
+
+    cv::Mat tmp;
+    // Test with one video source and vector
+    auto in_src = gapi::wip::make_src<cv::gapi::wip::GCaptureSource>(findDataFile("cv/video/768x576.avi"));
+    std::vector<int> first_in_vec(2304, 1);
```

Comment:
For clarity, I'd put "768*3" here instead of plain"2304"

Reply:
fixed